packages/cli/templates/oauth-callback.handlebars

```diff
@@ -1,10 +1,85 @@
 <html>
-	<script>
-		(function messageParent() {
-			const broadcastChannel = new BroadcastChannel('oauth-callback');
-			broadcastChannel.postMessage('success');
-		})();
-	</script>
-
-	Got connected. The window can be closed now.
+	<head>
+		<style>
+			html {
+				font-size: 16px;
+			}
+			body {
+				font-family: sans-serif;
+			}
+
+			.center-container {
+				display: flex;
+				align-items: center;
+				height: 100vh
+			}
+
+			.left-container {
+				margin-left: auto;
+				margin-right: auto;
+				display: flex;
+				flex-direction: column;
+			}
+			.row {
+				display: flex;
+				flex-direction: row;
+				gap: 20px;
+			}
+
+			.icon {
+				width: 2.5rem;
+				fill: #2AA568;
+			}
+
+			.logo {
+				width: 8rem;
+			}
+
+			h1 {
+				font-size: 2.5rem;
+				color: #0F1430;
+			}
+
+			p {
+				font-size: 1.5rem;
+				font-weight: 500;
+				color: #707183;
+				font-size: 1.1rem;
+			}
+		</style>
+	</head>
+	<body>
+		<div class="center-container">
+			<div class="left-container">
+				<div class="row">
+					<img src='https://github.com/n8n-io/n8n/blob/master/assets/n8n-logo.png?raw=true' class="logo" />
```

Comment:
Can we load this locally instead of pulling it from a remote service? If someone for some reason has GitHub blocked on an enterprise web filter or GitHub is having a hard time this could cause issues.